libraries/AP_Landing/AP_Landing_Deepstall.cpp

```diff
@@ -446,11 +465,15 @@ float AP_Landing_Deepstall::predict_travel_distance(const Vector3f wind, const f
 
     float estimated_forward = cosf(estimated_crab_angle) * forward_speed_ms + cosf(theta) * wind_length;
 
-#ifdef DEBUG_PRINTS
-    GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_INFO, "Predict: %f %f", stall_distance,  estimated_forward * height / down_speed + stall_distance);
-#endif // DEBUG_PRINTS
+    predicted_travel_distance = estimated_forward * height / down_speed + stall_distance;
```

Comment:
You know... I agree with everything you just said but this was all actually a preexisting problem (will still fix now of course). Did I not get you to review the Deepstall stuff originally for some reason? I feel like you would have pointed this out then :P